university-college-lillebaelt-vancouver.csl

```diff
@@ -270,7 +278,7 @@
   </citation>
   <bibliography et-al-min="7" et-al-use-first="6" second-field-align="flush">
     <layout>
-      <text variable="citation-number" suffix="."/>
+      <text variable="citation-number" suffix=". "/>
```

Comment:
That makes sense, though it's not ideal on the Zotero/citeproc-js side of things. @fbennett -- is this something citeproc does or do we need to look at the Word add-ons?